app/Providers/RouteServiceProvider.php

```diff
@@ -24,7 +24,7 @@ class RouteServiceProvider extends ServiceProvider
      *
      * When present, controller route declarations will automatically be prefixed with this namespace.
      *
-     * @var string|null
+     * @ var string|null
```

Comment:
Is this a typo in the comment?

Reply:
This is to prevent PHPStan from breaking on because the next line is commented

```
// protected $namespace = 'App\\Http\\Controllers';
```